app/assets/stylesheets/_variables.scss

```diff
@@ -0,0 +1,38 @@
+// ===================================
```

Comment:
@Mons-web Are this variables only for public module? If so we should keep them in personal directory (at web) so they will be isolated.